web/frontend/src/components/slam.vue

```diff
@@ -117,6 +117,30 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const fetchFeatureFlags = (name: string): Promise<{[key: string]: boolean}> => {
+  return new Promise((resolve, reject): void => {
+    const request = new commonApi.DoCommandRequest();
+    request.setName(name);
+    request.setCommand(Struct.fromJavaScript({ feature_flag: true }));
+    props.client.slamService.doCommand(
+      request,
+      new grpc.Metadata(),
+      (error: ServiceError|null, responseMessage: commonApi.DoCommandResponse|null) => {
+        if (error) {
```

Comment:
Please add a comment why we ignore the unimplemented error & reject for other errors.